src/tests/unit/tests/electron/platform/android/setup/android-setup-start-listener.test.ts

```diff
@@ -33,70 +28,26 @@ describe('AndroidSetupStartListener', () => {
         },
     );
 
-    it('sends ready-to-start when initial state is not ready but changed state is', () => {
```

Comment:
I wonder if we still want these test cases, just triggered with different changes w/o adbFeatureFlag

Reply:
Restored, triggering on a state change